File: Codigo/Obligatorio1DA2/Domain/Review.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Domain
{
    public class Review
    {
        public int Id { get; set; }
        public Lodging Lodging { get; set; }
        public string Description { get; set; }
        public int Score { get; set; }
        public Client Client { get; set; }

        public Review()
        {

        }

        public override bool Equals(object obj)
        {
            if (obj == null)
            {
                return false;
            }
            if (!(obj is Review))
            {
                return false;
            }
            return Lodging.Equals(((Review)obj).Lodging) &&
                Description.Equals(((Review)obj).Description) &&
                Client.Equals(((Review)obj).Client) &&
                Score.Equals(((Review)obj).Score);
        }
    }
}

File: Codigo/UruguayNatural/src/Models/Review.ts
import { Lodging } from './Lodging';

export class Review {
    lodging: Lodging;
    id: number;
    description: string;
    score: number;
    client: string;

    constructor(Description: string, Score: number, ReservationCode: string,
        Id: number, lodging: Lodging) {
        this.description = Description;
        this.score = Score;
        this.client = "";
        this.id= Id;
        this.lodging = lodging;
    }
}